python/xorbits/_mars/dataframe/groupby/tests/test_groupby_execution.py

```diff
@@ -1878,3 +1878,140 @@ def test_series_groupby_rolling_agg(setup, window, min_periods, center, closed,
     mresult = mresult.execute().fetch()
 
     pd.testing.assert_series_equal(presult, mresult.sort_index())
+
+
+def test_groupby_nth(setup):
```

Comment:
You can use `@pytest.mark.parametrize` to parametrize your params in `nth`, including:
- chunk_size: `None` and int value
- dropna: `None`、`any` and `all`
refer to https://docs.pytest.org/en/7.1.x/example/parametrize.html.
This can clean your code.